libopenage/curve/keyframe.h

```diff
@@ -27,16 +27,26 @@ class Keyframe {
 	 * New, default-constructed element at the given time
 	 */
 	Keyframe(const time::time_t &time) :
-		time{time} {}
+		timestamp{time} {}
 
 	/**
 	 * New element fron time and value
 	 */
 	Keyframe(const time::time_t &time, const T &value) :
-		time{time},
+		timestamp{time},
 		value{value} {}
 
-	const time::time_t time = time::TIME_MIN;
+	const time::time_t &time() const {
+		return this->timestamp;
+	}
+
+	const T &val() const {
+		return this->value;
+	}
+
+private:
+	time::time_t timestamp = time::TIME_MIN;
+
 	T value = T{};
```

Comment:
if one wants to change value in-place it should be no problem for the internal curve consistency. so having a function for it should not be necessary, too.

Reply:
Currently we are not doing that though, but I can make it public anyway.